Add rocksdb and trie-log subcommands to storage command (#1642)
hyperledger/besu-docs: docs/public-networks/reference/cli/subcommands.md

| TRIE_BRANCH_STORAGE            | 1885032116      | 138 GiB     | 138 GiB         | 0 B              |
| TRIE_LOG_STORAGE               | 267301          | 17 GiB      | 17 GiB          | 0 B              |
|--------------------------------|-----------------|-------------|-----------------|------------------|
| ESTIMATED TOTAL                | 4311820607      | 1104 GiB    | 337 GiB         | 767 GiB          |
|--------------------------------|-----------------|-------------|-----------------|------------------|
```

</TabItem>

</Tabs>

Displays the disk space used by the RocksDB key-value database, broken down into column families.

### `trie-log`

Provides Bonsai trie log related actions.

#### `count`

<Tabs>

<TabItem value="Syntax" label="Syntax" default>

```bash
besu --config-file <PATH-TO-CONFIG-FILE> storage trie-log count
```

</TabItem>

<TabItem value="Example" label="Example">

```bash
besu --config-file config.toml storage trie-log count
```

</TabItem>

<TabItem value="Example output" label="Example output">

```bash
trieLog count: 742311
 - canonical count: 681039
 - fork count: 217
 - orphaned count: 61055
```

</TabItem>

</Tabs>

> Counts the number of trie logs in the database. This is the number of keys for the `TRIE_LOG_STORAGE` column family in RocksDB. Canonical count represents the finalized blockchain, fork count represents non-finalized branches of the blockchain and orphaned count represents trie logs not in the blockchain which can happen during block creation.

Reviewer: ```suggestion
Counts the number of trie logs, i.e. the number of keys for the `TRIE_LOG_STORAGE` column family in the RocksDB:
- Canonical count represents the finalized blockchain
- Fork count represents non-finalized branches of the blockchain 
- Orphaned count represents trie logs not in the blockchain which can happen during block creation
```
